### PR TITLE
style(frontend): calculates amount for btc transactions correctly

### PR DESCRIPTION
**Pull request by BonomoAlessandro**
# Motivation

On the transactions page the BTC send transactions are displayed as if the user is receiving tokens instead of sending them. The amount of a `send` transaction should be displayed with a leading `-` (minus) and the text should be black.

# Changes

- fixes calculation

# Tests

**before:**
image

**after:**
image

### CODE REVIEW

File: src/frontend/src/btc/components/transactions/BtcTransaction.svelte [hunk 1]
```diff
@@ -21,11 +21,14 @@
 
 	let label: string;
 	$: label = type === 'send' ? $i18n.send.text.send : $i18n.receive.text.receive;
+
+	let amount: bigint | undefined;
+	$: amount = nonNullish(value) ? (type === 'send' ? value * BigInt(-1) : value) : undefined;
```
Comment (AntonioVentilii) [inline src/frontend/src/btc/components/transactions/BtcTransaction.svelte:26]: just to be sure, better use `value.mul(BigNumber.from(-1))` here, you never know with `BigInt`

File: src/frontend/src/tests/btc/components/transactions/BtcTransaction.spec.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+			transaction: mockTransaction,
+			token: DEFAULT_BITCOIN_TOKEN
+		};
+
+		const { getByText } = render(BtcTransaction, { props });
+
+		expect(getByText('Send')).toBeInTheDocument();
+		expect(getByText('-0.00040827')).toBeInTheDocument();
+	});
+	it('should calculate amount for receive transaction correctly', () => {
```
Comment (AntonioVentilii) [inline src/frontend/src/tests/btc/components/transactions/BtcTransaction.spec.ts:27]:
```suggestion

	it('should calculate amount for receive transaction correctly', () => {
```

**Review by AntonioVentilii — APPROVED** [review]
LGTM tks! Smaller comments before merging

File: src/frontend/src/btc/components/transactions/BtcTransaction.svelte (hunk 1 above)
Comment (BonomoAlessandro) [reply src/frontend/src/btc/components/transactions/BtcTransaction.svelte:26]: @AntonioVentilii `value` is of type `bigint` and does not provide a function `mul`.

Comment (AntonioVentilii) [reply src/frontend/src/btc/components/transactions/BtcTransaction.svelte:26]: ah! then all good! I thought it was `BigNumber`, my mistake, sorry

**Pull request merged**